.travis.yml

```diff
@@ -3,8 +3,8 @@ sudo: false
 
 language: go
 go:
-  - 1.6.4
-  - 1.7.4
+  - 1.9.2
+  - 1.8.5
```

Comment:
Wrong order, lower to higher (and the additions belong in #112).

Reply:
But it's travis.yml. You wrote about it in #110.